src/main/java/pl/javastart/task/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package pl.javastart.task;
+
+public class Rectangle {
+    private double rectangleA;
+    private double rectangleB;
```

Comment:
Jeżeli jesteśmy w klasie Rectangle, to nie ma potrzeby dodawania do nazw pól, czy metod "rect", albo "rectangle"